test/llvm/vectorization/parallel_loop.chpl

```diff
@@ -0,0 +1,15 @@
+
+proc loop (A, B) {
+  // This loop should get vectorized when passed --vectorize
+  // CHECK: <4 x i32>
+  for i in vectorizeOnly(0..511) {
+      A[A[i]] = B[i];
+      A[i] = B[i+1];
+  }
```

Comment:
Anyway I'd consider passing -vectorize-scev-check-threshold=0  -runtime-memory-check-threshold=0 LLVM options to disable the runtime check.